src/System.Management.Automation/engine/hostifaces/ConnectionBase.cs

```diff
@@ -685,7 +685,7 @@ public RunspaceEventQueueItem(RunspaceStateInfo runspaceStateInfo, RunspaceAvail
         }
 
         // This is to notify once runspace has been opened (RunspaceState.Opened)
-        internal ManualResetEventSlim RunspaceOpening = new ManualResetEventSlim(false);
+        private readonly ManualResetEventSlim _runspaceOpening = new ManualResetEventSlim(false);
```

Comment:
Made `private`, so that `ConnectionBase` has full control of the instance, without leaking it's implementation detail to other classes.